html/semantics/scripting-1/the-script-element/module/referrer-strict-policies.sub.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Referrer with the strict-origin referrer policy</title>
+<meta name="referrer" content="strict-origin">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script type="module">
+
+// "name" parameter is necessary for bypassing the module map.
+
+import { referrer as insecureImport } from "./resources/import-referrer-checker-insecure.sub.js?name=insecure_import";
+import { referrer as secureImport } from "https://{{domains[www]}}:{{ports[https][0]}}/html/semantics/scripting-1/the-script-element/module/resources/import-referrer-checker-insecure.sub.js?name=secure_import";
+
+const origin = (new URL(location.href)).origin + "/";
+
+test(t => {
+  assert_equals(
+      insecureImport, origin,
+      "A document with the strict-origin referrer policy served over HTTP, " +
+      "imports an module script over HTTPS, that imports a descendant script " +
+      "over HTTP. The request for the descendant script is sent with no " +
+      "`Referer` header");
```

Comment:
These messages seem to mismatch the asserts? I.e. this is testing that `insecureImport === origin` but the message says "the request for the descendant script is sent with _no_ `Referer` header"?

Reply:
Ah, must've switched these at some point